MIP/mip-34/README.md

```diff
@@ -0,0 +1,270 @@
+# MIP-34: Fast-Finality Settlement
+
+- **Description**: Establish the scope and components that are part of the Fast-Finality Settlement mechanism.
+- **Authors**: [Franck Cassez](), [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+
+## Abstract
+
+Fast-Finality Settlement (FFS) is a mechanism that allows for fast _confirmation_ of transactions backed by crypto-economic security. This MIP outlines the high-level specifications and architecture of FFS.
+
+## Definitions
+
+- **L2-block** - See [Overview](#overview)
+- **sequencer-batch** - See [Overview](#overview)
+- **block-range** - a sequence of blocks that are confirmed together. See [Overview](#overview)  
+- **validator** - a node that is responsible for validating transactions and confirming L2-blocks. See [Overview](#overview)  
```

Comment:
I probably am missing your point.. but just in case: 
I assume you referring towards rewards. In that case I dont think there should be any race. This would be bad as it incentivizes centralization and co-location. It also lowers the chain-quality as honest participants that dont make it regularly, will drop out. Nodes that submit votes (within a certain time interval) should get rewarded whether they have made it into the 2/3 or not. 

tbh i consider  "attester" to be the better term for "validator". But the debate of naming for this particiular entity seems out of scope for this MIP. just my 2c

Do you have a proposal for the justification for the term "validator"?